src/main/java/hudson/plugins/jacoco/ExecutionFileLoader.java

```diff
@@ -122,9 +121,8 @@ private IBundleCoverage analyzeStructure() throws IOException {
 				excludes = ITEM_ZERO;
```

Comment:
so this is the defaul exclude - but I have no idea why.  the default in maven in the empty String, but here we use `"{0}"` 🤷